src/locale/messages.fr.xlf

```diff
@@ -1,313 +1,365 @@
       <trans-unit datatype="html" id="baa5cda119114bb5fba61496780778a5fa10caf1">
         <source>Run test</source>
-        <target state="new">Lancer un test</target>
+        <target state="translated">Lancer un test</target>
       </trans-unit>
       <trans-unit datatype="html" id="0bd8b27f60a1f098a53e06328426d818e3508ff9">
         <source>Share</source>
-        <target state="new">Partager</target>
+        <target state="translated">Partager</target>
       </trans-unit>
       <trans-unit datatype="html" id="a119d6c78f3f291d3be8bac176f93afe0757f615">
         <source>Sorry, there is nothing here!</source>
-        <target state="new">Désolé, il n&apos;y a rien ici!</target>
+        <target state="translated">Désolé, il n'y a rien ici !</target>
       </trans-unit>
       <trans-unit datatype="html" id="2e52adeb40004b84f94197ec742951783fb4e377">
         <source>Display all results</source>
-        <target state="new">Afficher l&apos;ensemble des résultats</target>
+        <target state="translated">Afficher l'ensemble des résultats</target>
         <note priority="1" from="description">tooltip all</note>
       </trans-unit>
       <trans-unit datatype="html" id="688e4555813c90de96d11699aef24aab49d5b4af">
         <source>Critical - means a very serious error.</source>
-        <target state="new">Critical - une erreur vraiment sévère</target>
+        <target state="translated">Critique - une erreur vraiment sévère.</target>
         <note priority="1" from="description">tooltip critical</note>
       </trans-unit>
       <trans-unit datatype="html" id="fff721af9d16ca1cd0983fce54d10c5e857ba341">
         <source>Error - means a problem that is very likely (or possibly certain) to negatively affect the function of the zone being tested, but not so severe that the entire zone becomes unresolvable</source>
-        <target state="new">Error - une erreur qui a de grande change d&apos;affecter la zone testée, mais pas suffisamment pour la rendre inutilisable</target>
+        <target state="translated">Erreur - une erreur qui a de grande change d'affecter la zone testée, mais pas suffisamment pour la rendre inutilisable</target>
         <note priority="1" from="description">tooltip error</note>
       </trans-unit>
       <trans-unit datatype="html" id="281f2af4328daa895401b765738b1eb9cc55f489">
-        <source>Info - is something that may be of interest to the zone&apos;s administrator but that definitely does not indicate a problem.</source>
-        <target state="new">Info - contient des informations qui peuvent être d&apos;intérêt pour la personne lançant le test, ces messages ne sont pas des erreurs</target>
+        <source>Info - is something that may be of interest to the zone's administrator but that definitely does not indicate a problem.</source>
+        <target state="translated">Info - contient des informations qui peuvent être d'intérêt pour la personne lançant le test, ces messages ne sont pas des erreurs.</target>
```

Comment:
Remove trailing dot

Reply:
I disagree; the original sentence has a trailing full-stop, so the translation probably should too.

This particular translation could be significantly improved. How about: “Info – une information pouvant être utile pour l’administrateur de la zone mais qui n’est en aucun cas une anomalie.”

Reply:
> This particular translation could be significantly improved. 

Done